instances/UBERONParcellation/ansiformLobuleCrusII.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/ansiformLobuleCrusII",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "definition": "'Ansiform lobule crus II' is an ansiform lobule and hemisphere part of cerebellar posterior lobe.",

```

Comment:
Please change to "'Ansiform lobule crus II' is a hemisphere part of the cerebellar posterior lobe."

Reply:
Done.